Umbraco-Cloud/Cloud-API/index.md

```diff
@@ -46,13 +46,21 @@ To use this endpoint, make a request like this:
 
     {
         "projectName": "Name of the project", // Required
-        "plan": "Single", // Required. Options are: "Single or Studio"
+        "plan": "Single", // Required. Options are: "Single, Standard or Studio" Plan names are case sensitive!
         "ownerId": "c5821a98-ce88-4796-be90-e29f0a05fa39", // Optional. Can be a GUID of an organization or an email. If nothing is provided the owner becomes the user the token is associated with
         "baselineAlias": "an-alias-of-a-baseline" // Optional. If the project needs to be a child, then you can provide the alias of the baseline
     }
     
 :::note
-You'll notice the "plan" parameter takes either "Single" or "Studio" as valid options. The terminology used for these plans has since changed. "Single" will create a "Starter" plan whereas "Studio" a "Professional".
+You'll notice the "plan" parameter takes either "Single", "Standard" or "Studio" as valid options. The terminology used for these plans has since changed for the "Single" and "Studio" plans. "Single" will create a "Starter" plan whereas "Studio" a "Professional" one.
+:::
+
+:::note
+Do note that the /create endpoint can only be used for creating <b>v7</b> projects.
+:::
+
+:::note
+It's currently not possible to create Heartcore or Uno projects using the Cloud API. The Cloud API has not been updated for a longer time now, however, we're looking into doing a major makeover of the Cloud API.
```

Comment:
I think these two notes could be combined to one, and then added a bit further up in the article instead.

What do you think @RyuLindow ? 🤔

Reply:
Sure, I don't really mind. A bit hard to tell how notes look like without the actual preview 😐

Reply:
There's a good overview/preview of each of them in here: https://our.umbraco.com/documentation/Contribute/Markdown-Conventions/#adding-notes-warnings-tips

Reply:
Okay, when I said no actual preview, I meant that specific note. I know how they look like in general 😊

Reply:
Just merge when you can. I don't have anything more for that 😊